### PR TITLE
Add Windows CI legs

### PR DESCRIPTION
**Pull request by joperezr**
cc: @safern 

This will add Windows runs into our CI and official builds

FYI: @ulisesh @joshfree

### CODE REVIEW

File: .vsts-ci.yml [hunk 1]
```diff
@@ -54,6 +54,18 @@ jobs:
...
       displayName: Build
       condition: succeeded()
+    - script: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 -warnaserror:0 -ci
+        eng/sendToHelix.proj
+        /t:Test
+        /p:TestOS=Windows_NT
+        /p:Configuration=$(_BuildConfig)
+        /p:HelixBuild=$(Build.BuildNumber)
+        /bl:$(Build.SourcesDirectory)/artifacts/log/$(_BuildConfig)/SendToHelix.binlog
+        /p:HelixAccessToken=$(HelixApiAccessToken)
```
Comment (safern) [inline .vsts-ci.yml]: Will `HelixApiAccessToken` always be defined? What happens in PR builds were the variable group is not available?

File: eng/sendToHelix.proj [hunk 2]
```diff
@@ -16,36 +21,54 @@
     <FailOnMissionControlTestFailure>true</FailOnMissionControlTestFailure>
   </PropertyGroup>
 
-  <ItemGroup>
-    <!-- For now we only have a queue for Linux. Once we have a queue for Windows we will run 
-    those tests too. -->
+  <PropertyGroup Condition=" '$(HelixAccessToken)' == '' ">
+    <IsExternal>true</IsExternal>
```
Comment (safern) [inline eng/sendToHelix.proj]: I think `IsExternal` is no longer needed. I think the Helix SDK will set it to true if `HelixAccessToken` is empty

File: .vsts-ci.yml (hunk 1 above)
Comment (joperezr) [reply .vsts-ci.yml]: It is evaluated as empty. Which basically means that it won't be set.

Comment (safern) [reply .vsts-ci.yml]: Ok, cause I thought when a variable wasn't defined, azure devops would replace it with the literal `$(...)` value.

File: eng/sendToHelix.proj (hunk 2 above)
Comment (joperezr) [reply eng/sendToHelix.proj]: Ok, I'll check and if so I'll remove. Thanks

**Pull request merged**